vtex/toolbelt
# Create Manifest class to deal with manifest operations. Remove outdated commands

#### What is the purpose of this pull request?
- Create a Manifest class to concentrate all possible manifest operations
- Remove outdated commands: `vtex local manifest`, `vtex port react`, `vtex local package`

For now the new Manifest class isn't use in any commands. It needs to be validated by the reviewers first - new PRs are coming to use it on commands.

#### What problem is this solving?
This is a refactoring. The objective is to possibly get a more organized code on toolbelt to achieve more maintainability and code understandability.

#### How should this be manually tested?

Install this branch:
```
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout refactor/manifest-interface && \
yarn && yarn global add file:$PWD
```
Check if it compiles and passes all tests

#### Screenshots or example usage

#### Types of changes
